### PR TITLE
Drop duplicate release notes

### LINKED ISSUE #323: Duplicates in release notes
I saw almost every release note duplicated in one of our internal landscapes.
@petersutter can you check this please?

### PR DESCRIPTION
**Pull request by petersutter**
Fixes #323

### CODE REVIEW

File: github/release_notes/model.py [hunk 1]
```diff
@@ -28,19 +29,60 @@
     "is_current_repo",
     "from_same_github_instance",
     "cn_source_repo"
-])
+])):
+    def __eq__(self, other):
```
Comment (ccwienk) [inline github/release_notes/model.py]: why is this required? namedtuple instances should already have `__eq__` implementations with this sematics (equal iff all attrs are equal).

also, consider switching to data classes (new feature in python3.7):

https://docs.python.org/3/library/dataclasses.html

File: github/release_notes/model.py [hunk 2]
```diff
@@ -28,19 +29,60 @@
...
+
 
-Commit = namedtuple('Commit', [
+class Commit(namedtuple('Commit', [
     "hash",
     "subject",
     "message"
-])
+])):
+    def __eq__(self, other):
```
Comment (ccwienk) [inline github/release_notes/model.py]: also: should not be required.

File: github/release_notes/model.py [hunk 3]
```diff
@@ -55,10 +97,19 @@
 )
 REFERENCE_TYPES = [REF_TYPE_PULL_REQUEST, REF_TYPE_COMMIT]
 
-Reference = namedtuple('Reference', [
+
+class Reference(namedtuple('Reference', [
```
Comment (ccwienk) [inline github/release_notes/model.py]: ditto

File: github/release_notes/renderer.py [hunk 4]
```diff
@@ -77,7 +77,7 @@ def get_or_call(obj, path):
 
 class Renderer(object):
     def __init__(self, release_note_objs: [ReleaseNote]):
-        self.rls_note_objs = release_note_objs
+        self.rls_note_objs = _.uniq(release_note_objs)
```
Comment (ccwienk) [inline github/release_notes/renderer.py]: i assume this single change should be sufficient for the intended fix

**Comment by petersutter** [comment]
yes correct, thanks. I have addressed your comments and also went with `dataclasses`. PTAL

**Comment by rfranzke** [comment]
ping
